### PR TITLE
Fixes Issue #42 and #45 - Allow users to enlarge images in player and allow feedback for correct answers

### PR DESCRIPTION
**Pull request by cayb0rg**

### CODE REVIEW

File: src/player.html [hunk 1]
```diff
@@ -184,7 +208,7 @@ <h1>{{gameState.splashtext}}</h1>
 			<section class="lightbox" ng-class="{ 'show': lightboxTarget != -1}" ng-hide="lightboxTarget == -1" ng-click="setLightboxTarget(-1)">
 				<div class="lightbox-content">
 					<button class="lightbox-close" ng-click="setLightboxTarget(-1)">X</button>
-					<iframe class="lightbox-video" ng-if="answers[lightboxTarget].options.asset.type == 'video'" ng-src="{{answers[lightboxTarget].options.asset.value}}" frameborder="0" allowfullscreen></iframe>
+					<iframe class="lightbox-iframe" ng-if="answers[lightboxTarget].options.asset.type == 'video' || answers[lightboxTarget].options.asset.type == 'image'" ng-src="{{answers[lightboxTarget].options.asset.value}}" frameborder="0" allowfullscreen></iframe>
```
Comment (clpetersonucf) [inline src/player.html:211]: Like I mentioned in my review comments, this should probably be a `div` instead of an `iframe` for the image version.

**Review by clpetersonucf — CHANGES_REQUESTED** [review]
That was quick - and easy! The changes look great, here are four small follow-ups I'd like to see included in this pull request as well:
- The non-expanded version of images are still being stretched to fill the container, instead of having their aspect ratios preserved. This isn't apparent in the dev kit since all the images provided have aspect ratios that fit the container. You'll probably have to install the widget to your local copy of Materia to test this out.
- Images in the expanded lightbox should be centered instead of left-justified
- The lightbox for images doesn't have to be an iframe - we're only doing that for video because its embedding content from YouTube or Vimeo
- The instructional Designer who requested these changes was also interested in optional feedback for _correct_ answers as well as incorrect answers. This will require modifying the creator as well as the question set (qset) structure for the widget, which requires some care. I'm happy to go over this in more detail.

**Review by ljoks — CHANGES_REQUESTED** [review]
Really nice job overall, especially catching the backwards compatibility! 
Two things I noticed that need fixing: 

1. Although the fields say feedback is an optional field (for both correct and incorrect), trying to save while leaving them blank causes an error. see `src/services/srv-creator.js` lines 44 & 58
2. When you go to update an existing widget, feedback values are not populated in the fields for correct or incorrect

**Review by clpetersonucf — CHANGES_REQUESTED** [review]
I can confirm the behaviors LJ mentioned. In addition, I'm also noticing that while the player preserves image aspect ratios, the creator does not. Ideally, the aspect ratios are preserved in both, so images don't look stretched or distorted to the user as soon as they upload them.

**Review by clpetersonucf — APPROVED** [review]
This looks good and works great. I ended up adding the backwards-compatibility checks for the old feedback location, just because they were small changes.

**Pull request merged**